src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -320,14 +321,26 @@ public void addRow(T item, List<Content> contents) {
             }
         }
         int colIndex = 0;
+        Pattern pattern = Pattern.compile("<a\\b");
         for (Content c : contents) {
             HtmlStyle cellStyle = columnStyles.get(colIndex);
+            // Only make element tabbable if it doesn't contain tabbable content to avoid widget_tabbable_single(Potential Violation)
+            // For more info refer : https://able.ibm.com/rules/archives/latest/doc/en-US/widget_tabbable_single.html
+            boolean matchFound = pattern.matcher(c.toString()).find();
             // Always add content to make sure the cell isn't dropped
             var cell = HtmlTree.DIV(cellStyle).addUnchecked(c.isEmpty() ? Text.EMPTY : c);
             cell.addStyle(rowStyle);
+            if (!matchFound) {
+                cell.put(HtmlAttr.ROLE, "tablist")
```

Comment:
Looking into this further, I don't think the `role="tablist"` attribute is correct here. According to the [MDN documentation], a `tablist` is defined as a container for a set of `tabs`, which does not apply for table cells.